### PR TITLE
[Network] skip second connect in bidirectional peering

### PR DESCRIPTION
**Pull request by cheina97**
# Description

This PR disable a second connection establishment in case of a bidirectional peering.

### CODE REVIEW

File: pkg/liqoctl/network/handler.go [hunk 1]
```diff
@@ -63,6 +64,39 @@ func NewOptions(localFactory *factory.Factory) *Options {
...
+	} else if client.IgnoreNotFound(err) != nil {
+		return false, err
+	}
+
+	_, err = cluster2.GetGatewayClient(ctx, forge.DefaultGatewayServerName(cluster1.localClusterID))
+	if err == nil {
+		return true, nil
+	} else if client.IgnoreNotFound(err) != nil {
+		return false, err
+	}
```
Comment (aleoli) [inline pkg/liqoctl/network/handler.go:95]: I would check for both client and server on both clusters

**Review by aleoli — APPROVED** [review]

**Review by fra98 — APPROVED** [review]
I think it's better to handle the check inside the liqoctl network commands (i.e., `network connect` and if necessary `network init`), and not on the peering command, so that also `network init/connect` works too.

File: pkg/liqoctl/network/handler.go (hunk 1 above)
Comment (cheina97) [reply pkg/liqoctl/network/handler.go:95]: It would become impossible to change network parameters using liqoctl peer without unpeer

**Review by fra98 — APPROVED** [review]

**Pull request merged**